Helm Release Panic "Helm uninstall returned information: %q'"

## Issue details

Running up that would uninstall a Release ends up with a panic.

```
 -   └─ kubernetes:helm.sh/v3:Release  rancher-monitoring  deleting...  
panic: fatal: An assertion has failed: Urn is: 'Helm uninstall returned information: %q'

goroutine 623 [running]:
github.com/pulumi/pulumi/sdk/v3/go/common/util/contract.failfast(...)
        /private/tmp/pulumi-20211110-2384-1rql7ko/sdk/go/common/util/contract/failfast.go:23
github.com/pulumi/pulumi/sdk/v3/go/common/util/contract.Assertf(0xe0, {0x56c8bd4, 0x40123d4}, {0xc002c97640, 0xc002c97650, 0x4017ae2})
        /private/tmp/pulumi-20211110-2384-1rql7ko/sdk/go/common/util/contract/assert.go:33 +0xf4
github.com/pulumi/pulumi/sdk/v3/go/common/resource.URN.URNName({0xc00205c4e0, 0x27})
        /private/tmp/pulumi-20211110-2384-1rql7ko/sdk/go/common/resource/urn.go:82 +0x8c
github.com/pulumi/pulumi/sdk/v3/go/common/resource.URN.Type({0xc00205c4e0, 0xc003162510})
        /private/tmp/pulumi-20211110-2384-1rql7ko/sdk/go/common/resource/urn.go:103 +0x25
github.com/pulumi/pulumi/pkg/v3/backend/display.isRootURN(...)
        /private/tmp/pulumi-20211110-2384-1rql7ko/pkg/backend/display/display.go:152
github.com/pulumi/pulumi/pkg/v3/backend/display.(*ProgressDisplay).getRowForURN(0xc00076e780, {0xc00205c4e0, 0x560d260}, 0x0)
        /private/tmp/pulumi-20211110-2384-1rql7ko/pkg/backend/display/progress.go:988 +0x145
github.com/pulumi/pulumi/pkg/v3/backend/display.(*ProgressDisplay).processNormalEvent(0xc00076e780, {{0x56ba735, 0x4}, {0x560d260, 0xc001edaf00}})
        /private/tmp/pulumi-20211110-2384-1rql7ko/pkg/backend/display/progress.go:1085 +0x807
github.com/pulumi/pulumi/pkg/v3/backend/display.(*ProgressDisplay).processEvents(0x4, 0xc000dd6190, 0xc002590600)
        /private/tmp/pulumi-20211110-2384-1rql7ko/pkg/backend/display/progress.go:1203 +0xe5
github.com/pulumi/pulumi/pkg/v3/backend/display.ShowProgressEvents.func1()
        /private/tmp/pulumi-20211110-2384-1rql7ko/pkg/backend/display/progress.go:343 +0x2e
created by github.com/pulumi/pulumi/pkg/v3/backend/display.ShowProgressEvents
        /private/tmp/pulumi-20211110-2384-1rql7ko/pkg/backend/display/progress.go:342 +0x7b6
```

Pulumi Version: v3.17.1
Language: Python
Pulumi Kubernetes Library Version: v3.10.0


### Steps to reproduce
1. Install rancher-monitoring chart using Release by running up
2. Comment out code
3. Run up to uninstall release

Expected: Release is uninstalled
Actual: Panics with error "Helm uninstall returned information: %q'"

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,7 @@
 ## HEAD (Unreleased)
 
 - Remove unused helm ReleaseType type (https://github.com/pulumi/pulumi-kubernetes/pull/1805)
+- Fix Helm Release Panic "Helm uninstall returned information" (https://github.com/pulumi/pulumi-kubernetes/pull/1807)
 
 ## 3.10.0 (November 12, 2021)
 

--- a/provider/pkg/provider/helm_release.go
+++ b/provider/pkg/provider/helm_release.go
@@ -976,7 +976,7 @@ func (r *helmReleaseProvider) Delete(ctx context.Context, req *pulumirpc.DeleteR
 	}
 
 	if res.Info != "" {
-		_ = r.host.Log(context.Background(), diag.Warning, "Helm uninstall returned information: %q", res.Info)
+		_ = r.host.Log(context.Background(), diag.Warning, urn, fmt.Sprintf("Helm uninstall returned information: %q", res.Info))
 	}
 	return &pbempty.Empty{}, nil
 }

